locale/core.yml

```diff
@@ -238,8 +238,11 @@ core:
 
     # These translations are used in the Edit User modal dialog (admin function).
     edit_user:
+      activate: Activate User
```

Comment:
This key name needs a suffix to indicate how the string is used (e.g., `_button`, `_label`).

Please refer to the [Appendix of the i18n doc](http://flarum.org/docs/extend/internationalize/#naming-id-keys) for details.